flink-ai-flow/ai_flow/java/client/src/main/java/com/aiflow/client/AIFlowClient.java

```diff
@@ -79,63 +78,57 @@ public AIFlowClient(
   }
 
   /**
-   * Get a specific example in Metadata Store by example id.
+   * Get a specific dataset in Metadata Store by dataset id.
    *
-   * @param exampleId Id of example.
-   * @return Single ExampleMeta object if example exists, otherwise returns None if example does not
+   * @param datasetId Id of dataset.
+   * @return Single DatasetMeta object if dataset exists, otherwise returns None if dataset does not
    *     exist.
    */
-  public ExampleMeta getExampleById(Long exampleId) throws Exception {
-    return this.metadataClient.getExampleById(exampleId);
+  public DatasetMeta getDatasetById(Long datasetId) throws Exception {
+    return this.metadataClient.getDatasetById(datasetId);
   }
 
   /**
-   * Get a specific example in Metadata Store by example name.
+   * Get a specific dataset in Metadata Store by dataset name.
    *
-   * @param exampleName Name of example.
-   * @return Single ExampleMeta object if example exists, otherwise returns None if example does not
+   * @param datasetName Name of dataset.
+   * @return Single DatasetMeta object if dataset exists, otherwise returns None if dataset does not
    *     exist.
    */
-  public ExampleMeta getExampleByName(String exampleName) throws Exception {
-    return this.metadataClient.getExampleByName(exampleName);
+  public DatasetMeta getDatasetByName(String datasetName) throws Exception {
+    return this.metadataClient.getDatasetByName(datasetName);
   }
 
   /**
-   * Register a example in Metadata Store.
+   * Register a dataset in Metadata Store.
    *
-   * @param name Name of example.
-   * @param supportType Example's support type.
-   * @param dataFormat Data format of example.
-   * @param description Description of example.
-   * @param batchUri Batch uri of example.
-   * @param streamUri Stream uri of example.
-   * @param createTime Time when example is created.
-   * @param updateTime Time when example is updated.
-   * @param properties Properties of example.
-   * @param nameList Name list of example's schema.
-   * @param typeList Type list corresponded to name list of example's schema.
-   * @return Single ExampleMeta object registered in Metadata Store.
+   * @param name Name of dataset.
+   * @param dataFormat Data format of dataset.
+   * @param description Description of dataset.
+   * @param uri Uri of dataset.
+   * @param createTime Time when dataset is created.
+   * @param updateTime Time when dataset is updated.
```

Comment:
`createTime` and `updateTime` don't need to define as parameters for `registerDataset`.